src/test/0_Template.t.sol

```diff
@@ -1,11 +1,8 @@
 // SPDX-License-Identifier: MIT
-pragma solidity 0.8.14;
+pragma solidity 0.8.15;
```

Comment:
WDYT about just doing this? IDK why I went with 14 in the first place, and if the point is to just get code running quickly, we should prefer flexibility. 

```suggestion
pragma solidity ^0.8.0;
```